<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>module2</class>
 <widget class="QWidget" name="module2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backModuleS">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#backModuleS{
image: url(:/assets/iconos/left.png);
background: transparent;
}
#backModuleS:hover{
	image: url(:/assets/iconos/leftH.png);
}
#backModuleS:pressed{
	image: url(:/assets/iconos/leftP.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nextPart">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>10</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#nextPart{
image: url(:/assets/iconos/test.png);
background: transparent;
}
#nextPart:hover{
	image: url(:/assets/iconos/testH.png);
}
#nextPart:pressed{
	image: url(:/assets/iconos/testP.png);
}
#nextPart:disabled{
	image: url(:/assets/iconos/testD.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="partText">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>850</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="buddy">
    <cstring>nextPart</cstring>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>800</width>
     <height>500</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="page0">
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color: transparent;
border-image: url(:/assets/modules/m2p1.png);
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="page1">
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color: transparent;
border-image: url(:/assets/modules/m2p2.png);
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="page2">
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color: transparent;
border-image: url(:/assets/modules/m2p3.png);
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="page3">
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color: transparent;
border-image: url(:/assets/modules/m2p4.png);
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="page4">
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color: transparent;
border-image: url(:/assets/modules/m2p5.png);
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="stack0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>130</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: white;
border: 2px solid black;
border-radius: 10px;
}


QPushButton:hover{
border: 2px solid #ebae13;
}</string>
   </property>
   <property name="text">
    <string>Conjuntos y
 elementos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stack1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>131</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: white;
border: 2px solid black;
border-radius: 10px;
}


QPushButton:hover{
border: 2px solid #ebae13;
}</string>
   </property>
   <property name="text">
    <string>Notación de
 conjuntos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stack2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>131</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: white;
border: 2px solid black;
border-radius: 10px;
}


QPushButton:hover{
border: 2px solid #ebae13;
}</string>
   </property>
   <property name="text">
    <string>Unión (⋃) e 
Intersección (⋂)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stack3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>131</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: white;
border: 2px solid black;
border-radius: 10px;
}


QPushButton:hover{
border: 2px solid #ebae13;
}</string>
   </property>
   <property name="text">
    <string>Diferencia(–) y
Complemento(')</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stack4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>131</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: white;
border: 2px solid black;
border-radius: 10px;
}


QPushButton:hover{
border: 2px solid #ebae13;
}</string>
   </property>
   <property name="text">
    <string>Diferencia
simetrica (∆)</string>
   </property>
  </widget>
  <zorder>stackedWidget</zorder>
  <zorder>backModuleS</zorder>
  <zorder>nextPart</zorder>
  <zorder>partText</zorder>
  <zorder>stack0</zorder>
  <zorder>stack1</zorder>
  <zorder>stack2</zorder>
  <zorder>stack3</zorder>
  <zorder>stack4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
